aikau/src/main/resources/alfresco/renderers/InlineEditProperty.js

```diff
@@ -476,6 +541,8 @@ define(["dojo/_base/declare",
        */
       onSave: function alfresco_renderers_InlineEditProperty__onSave(formPayload) {
          /*jshint unused:false*/
+         domClass.add(this.domNode, "alfresco-renderers-InlineEditProperty--updating");
```

Comment:
Discussed offline. Agreed to leave this for now as it's not been problematic in testing. Also, in the majority of cases we don't actually want it to display as it'll be so fast.